src/Skeleton/PostCreateProject.php

```diff
@@ -360,6 +360,8 @@ private static function reconfigureApplication(Event $event): void
 MAILER_DSN=smtp://127.0.0.1:1025
 ###< symfony/mailer ###
 
+DATABASE_URL="mysql://root:root@127.0.0.1:3306/db_name_replace_me"
```

Comment:
In theorie zou je die kunnen weten op basis van client en project.
Maar vermoedelijk zit dat in `sumo`-command om deploy.php enzo aan te passen.